pyanaconda/payload/tarpayload.py

```diff
@@ -1,85 +0,0 @@
-# tarpayload.py
```

Comment:
This is strange - I think this should have been one of the lesser known but still supported payloads. Also IIRC the talk by a Yahoo employee about sub minute baremetal provisioning a couple years on Flock in Prague was IIRC using the tar payload. Might be good to investigate if there is really no way to use a tar payload with Anaconda and decide what to do about that, possibly asking Composer as well.

Reply:
I cannot find any proof that `TarPayload` was ever used in Anaconda. Also, notice the comment `TarPayload is not yet fully implemented`.

Btw. the `liveimg` command supports tarfiles:
https://pykickstart.readthedocs.io/en/latest/commands.html?highlight=liveimg#liveimg

Reply:
I'm also not convinced about removing this. It is weird to have it here. I want to look on this more before the removal.

Reply:
After some more investigation it looks like tar payload was implemented in the Great RHEL 7 rewrite as some first version and from that time no-one touch that. So I think it is safe to remove this, especially when this functionality is already implemented in the liveimg payload.